src/documents/tests/test_document_model.py

```diff
@@ -56,34 +56,43 @@ def test_file_name(self):
         doc = Document(
             mime_type="application/pdf",
             title="test",
-            created=timezone.datetime(2020, 12, 25),
+            created=timezone.datetime(2020, 12, 25, tzinfo=zoneinfo.ZoneInfo("UTC")),
         )
         self.assertEqual(doc.get_public_filename(), "2020-12-25 test.pdf")
 
+    @override_settings(
+        TIME_ZONE="Europe/Berlin",
+    )
     def test_file_name_with_timezone(self):
 
+        # See https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.timezone.now
+        # The default for created is an aware datetime in UTC
+        # This does that, just manually, with a fixed date
+        local_create_date = timezone.datetime(
+            2020,
+            12,
+            25,
+            tzinfo=zoneinfo.ZoneInfo("Europe/Berlin"),
+        )
+
+        utc_create_date = local_create_date.astimezone(zoneinfo.ZoneInfo("UTC"))
+
+        self.assertEqual(utc_create_date.date().day, 24)
```

Comment:
Can you move this to the other assert? Not so sure if the project follows [GivenWhenThen](https://martinfowler.com/bliki/GivenWhenThen.html) officially, but I think esp. with growing test code base DEVs can review tests more easy.

Reply:
That's an interesting way of thinking about test scenarios.   I like it.  I moved this one down with the other in the case.